docs/docs/Components/components-tools.md

```diff
@@ -66,14 +111,13 @@ The `Astra DB Tool` allows agents to connect to and query data from Astra DB col
 
 ### Outputs
 
-The Data output is primarily used when directly querying Astra DB, while the Tool output is used when integrating with LangChain agents or chains.
+The **Data** output is used when directly querying Astra DB, while the **Tool** output is used when integrating with agents.
 
 | Name | Type | Description |
 |------|------|-------------|
 | Data | List[`Data`] | A list of [Data](/concepts-objects) objects containing the query results from Astra DB. Each `Data` object contains the document fields specified by the projection attributes. Limited by the `number_of_results` parameter. |
```

Comment:
If this is how the tool operates and what it returns then nothing for you to change! I had thought that they were removing all List[`Data`] outputs... but maybe hasn't been done for this yet.

Reply:
It's still Data AFAICT https://github.com/langflow-ai/langflow/blob/main/src/backend/base/langflow/components/tools/astradb.py#L412